### PR TITLE
Add JP60 keyboard

### PR DESCRIPTION
**Pull request by ai03-2725**
## Description

Adds support for the [JP60](https://github.com/ai03-2725/JP60) open-source JIS keyboard kit.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* Adds support for JP60

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/ai03/jp60/readme.md
Comment (zvecr) [inline keyboards/ai03/jp60/readme.md:13]:
```suggestion
    make ai03/jp60:default
```

File: keyboards/ai03/jp60/readme.md
Comment (zvecr) [inline keyboards/ai03/jp60/readme.md:17]:
```suggestion
    make ai03/jp60:default:flash
```

**Review by zvecr — CHANGES_REQUESTED** [review]

File: keyboards/ai03/jp60/readme.md
Comment (ai03-2725) [reply keyboards/ai03/jp60/readme.md]: Whoops. No clue how I missed this.
Fixed with [e558964](https://github.com/qmk/qmk_firmware/pull/14219/commits/e558964fd92bbf655aabe4414929ed96d569a4bb).

File: keyboards/ai03/jp60/readme.md
Comment (ai03-2725) [reply keyboards/ai03/jp60/readme.md]: Resolved with [e558964](https://github.com/qmk/qmk_firmware/pull/14219/commits/e558964fd92bbf655aabe4414929ed96d569a4bb) as well.

File: keyboards/ai03/jp60/keymaps/default/config.h
Comment (drashna) [inline keyboards/ai03/jp60/keymaps/default/config.h:21]:
```suggestion
#define QMK_KEYS_PER_SCAN 12

```

File: keyboards/ai03/jp60/keymaps/via/config.h
Comment (drashna) [inline keyboards/ai03/jp60/keymaps/via/config.h:21]:
```suggestion
#define QMK_KEYS_PER_SCAN 12

```

File: keyboards/ai03/jp60/rules.mk
Comment (drashna) [inline keyboards/ai03/jp60/rules.mk:10]:
```suggestion
BOOTMAGIC_ENABLE = yes      # Enable Bootmagic Lite
```

**Review by drashna — CHANGES_REQUESTED** [review]

**Review by drashna — APPROVED** [review]

**Comment by ai03-2725** [comment]
Wanted to follow up just in case:
Is there something I still need to do for having the PR approved?

File: keyboards/ai03/jp60/info.json
Comment (drashna) [inline keyboards/ai03/jp60/info.json:6]:
```suggestion
```

**Review by drashna — CHANGES_REQUESTED** [review]

**Comment by drashna** [comment]
> Wanted to follow up just in case: Is there something I still need to do for having the PR approved?

Mostly, just needs a second review approval.

File: keyboards/ai03/jp60/rules.mk
Comment (fauxpark) [inline keyboards/ai03/jp60/rules.mk:21]:
```suggestion
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by fauxpark — APPROVED** [review]

**Review by drashna — APPROVED** [review]
Thanks!

**Pull request merged**
